services/movie_session.py

```diff
@@ -42,3 +42,19 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    taken_seats = []
+
+    tickets = Ticket.objects.filter(movie_session__id=movie_session_id)
+
+    for ticket in tickets:
+        taken_seats.append(
+            {
+                "row": ticket.row,
+                "seat": ticket.seat
+            }
+        )
+
+    return taken_seats
```

Comment:
You cna use list comprehension here

Reply:
You can try it, also a good solution. But it's ok to leave regular loop if you think it's more clear